EMS.py

```diff
@@ -534,15 +545,21 @@ def make_pairs_df(ems_list, write=False, max_pathlen=6):
 
 
 file_dir = './tests/test_mols/'
-file = 'testmol_1_NMR.nmredata.sdf'
+file = 'testmol_4_NMR.nmredata.sdf'
 path = file_dir + file
 
 mol = EMS(path, mol_id = file, fragment = True)
 print(mol.type)
 print(mol.xyz[:, 0].shape)
 print(mol.symmetric)
-print(mol.H_index)
-print(mol.H_to_reduce)
+print(mol.H_index_dict)
+print(mol.reduced_H_dict)
+print(mol.reduced_H_list)
+print(mol.eff_atom_list)
+print(mol.dumb_atom_list)
+# print(mol.reduced_conn[:, 47])
+# print(mol.conn[:, 47])
+print(len(mol.type))
```

Comment:
Move this into a test_EMS.py script under the tests folder to avoid cluttering the class and for functionality with Pytest